autoload/fern/scheme/file/mapping.vim

```diff
@@ -205,6 +207,28 @@ function! s:map_remove(helper) abort
         \.then({ -> a:helper.sync.echo(printf('%d items are removed', len(ps))) })
 endfunction
 
+function! s:map_preview(helper, mode) abort
+  let node = a:helper.sync.get_cursor_node()
+  if node is# v:null
+    return s:Promise.reject('cursor node is not visible')
+  endif
+  if node.status isnot# a:helper.STATUS_NONE
+    return s:Promise.reject('cannot preview a directory')
+  endif
+
+  try
+    execute "noautocmd bo pedit " .. node._path
+
+    if a:mode ==# 'jump'
+      wincmd P
+    endif
```

Comment:
I'm not 100% sure but I think we can write mapping like below to jump (please confirm if it's true.)

```vim
nnoremap P <Plug>(fern-action-preview)<C-w>p
```